src/components/ShareIcon/index.tsx

```diff
@@ -13,7 +13,7 @@ interface ShareButtonIconProps {
 
 const ShareButtonIcon: React.FC<ShareButtonIconProps> = ({
   onPress,
-  fill = COLORS.black,
+  fill = colors.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default fill color for the `ShareButtonIcon` component has been changed from `COLORS.black` to `colors.black`. As with the import statement, ensure that `colors.black` is defined in the `colors` module.
```diff
-  fill = COLORS.black,
+  fill = colors.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->